testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/AbstractSinglenodeDeltaLakeDatabricks.java

```diff
@@ -61,6 +61,8 @@ public void extendEnvironment(Environment.Builder builder)
                 .withEnv("DATABRICKS_TEST_TOKEN", databricksTestToken)
                 .withCopyFileToContainer(forHostPath(configDir.getPath("hive.properties")), CONTAINER_PRESTO_ETC + "/catalog/hive.properties")
                 .withCopyFileToContainer(forHostPath(configDir.getPath("delta.properties")), CONTAINER_PRESTO_ETC + "/catalog/delta.properties"));
+        builder.addConnector("hive");
```

Comment:
pls use `io.trino.tests.product.launcher.env.Environment.Builder#addConnector(java.lang.String, org.testcontainers.utility.MountableFile, java.lang.String)` method (and remove along the way the `.withCopyFileToContainer` calls from the lines above).